docs/getting-started/security-ui.asciidoc

```diff
@@ -16,210 +17,110 @@ image::images/search-bar.png[]
 
 [role="screenshot"]
-image::detections/images/all-rules.png[Shows the Rules page]
+image::images/timeline-ui.png[Shows the Timeline page]
+
+TIP: Select the collapsable *Timeline* button at the bottom of the {security-app} to start an investigation.
 
 [float]
-[[exceptions-page]]
-=== Exception lists page
+=== Cases
 
-The Exception lists page allows you to view and manage all rule exceptions. Refer to <<detections-ui-exceptions, Rule exceptions and value lists>> for more information about rule exceptions.
+Open and track security issues. Refer to <<cases-overview>> to learn more.
 
 [role="screenshot"]
-image::detections/images/exceptions-page.png[Shows the Exception lists page]
+image::cases/images/cases-home-page.png[Shows the Cases page]
 
 [float]
-[[hosts-ui]]
-=== Hosts page
+=== Explore
 
-The Hosts page provides key metrics for host-related security events and a set of interactive data tables. Refer to <<hosts-overview, Hosts page>> for more information.
+Expand this section to view the following pages:
 
+* <<hosts-overview, *Hosts*>>: Examine key metrics for host-related security events using graphs, charts, and interactive data tables.
++
 [role="screenshot"]
 image::management/hosts/images/hosts-ov-pg.png[Shows the Hosts page]
-
-
-[float]
-[[network-page]]
-=== Network page
-
-The Network page provides key network activity metrics via an interactive map and network event tables
-that enable interaction with Timeline. Refer to <<network-page-overview, Network page overview>> for more information.
-
++
+* <<network-page-overview, *Network*>>: Explore the interactive map to discover key network activity metrics and investigate network events further in Timeline.
++
 [role="screenshot"]
 image::images/network-ui.png[Shows the Network page]
-
-[float]
-[[users-page-ov]]
-=== Users page
-
-The Users page provides a comprehensive overview of user data to help you understand authentication and user behavior within your environment. Refer to <<users-page, Users page>> for more information.
++
+* <<users-page, *Users*>>: Access a comprehensive overview of user data to help you understand authentication and user behavior within your environment.
 
 [role="screenshot"]
 image::images/users/users-page.png[Shows the Users page]
 
-
 [float]
-[[timelines-page]]
-=== Timelines page
-
-Use the Timelines page to investigate alerts and complex threats, such as lateral movement of malware across hosts in your network. Timelines are responsive and allow you to share your findings among other team members. Refer to <<timelines-ui, Investigate events in Timeline>> for information about getting started with Timelines.
+=== Get started
 
-TIP: Select the collapsable *Timeline* button at the bottom of the {security-app} to start an investigation.
+View a quick shortcut to add security integrations, which you can use to monitor your hosts and ingest data.
```

Comment:
```suggestion
Add security integrations that can ingest data and monitor your hosts.
```

Reply:
Another possible revison that keeps a bit more of the original sentence:

```suggestion
Quickly add security integrations, which you can use to monitor your hosts and ingest data.
```